components/slider/Slider.razor.cs

```diff
@@ -320,17 +340,56 @@ private double RightValue
         [Parameter]
         public Action<TValue> OnChange { get; set; }
 
+        [Parameter]
+        public bool HasTooltip { get; set; } = true;
```

Comment:
I decided to move from the logic of having `TipFormatter` as null to indicate that the tooltip should not be shown at all. My problem here was that I could not pass null in the markup
```razor
<Slider ... TipFormatter="null"/>
```
(and other modifications) were showing compilation errors.